src/main/java/net/superdark/minecraft/plugins/SuperDarkCore/registration/AbstractPluginRegistration.java

```diff
@@ -0,0 +1,19 @@
+package net.superdark.minecraft.plugins.SuperDarkCore.registration;
+
+import net.superdark.minecraft.plugins.SuperDarkCore.SuperDarkCorePlugin;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public abstract class AbstractPluginRegistration extends JavaPlugin
```

Comment:
We should name this something like BaseSuperDarkPlugin not AbstractPluginRegistration.